R/robin_lm.R

```diff
@@ -37,6 +36,8 @@ robin_lm <- function(
       without treatment-covariate interactions; see the 2023 FDA guidance."
     )
   }
+  fit <- lm(formula, data = data, ...)
+  pc <- eval(bquote(predict_counterfactual(fit, treatment, data, vcov = .(substitute(vcov)), vcov_args = vcov_args)))
```

Comment:
maybe worth to add a comment here why we need to handle this with `eval/bquote` etc and not a simple call as in `robin_glm`